docs/rules/ultiduo.md

```diff
@@ -24,10 +24,19 @@ Failure to observe weapon whitelisting will result in league penalties. For Ulti
 - **Medic**
     - Syringe Gun & Syringe Gun Reskins
     - Crusader's Crossbow
+ 
+### 1.3. Weapon Pickups
+Weapons dropped on the ground by dead players are **allowed** to be picked up by surviving players - including the medigun.
```

Comment:
Would read better as "Dropped weapons can be picked up - including any medigun."